tests/integration/test_load_and_run_agent.py

```diff
@@ -96,9 +98,19 @@ def write_file(text: str) -> None:
         assert content == str(datetime.now().year)
         assert isinstance(agent_trace, AgentTrace)
         assert agent_trace.final_output
+        update_trace = request.config.getoption("--update-trace-assets")
+        if update_trace and _is_tracing_supported(agent_framework):
+            trace_path = (
+                Path(__file__).parent.parent
```

Comment:
before, I was using hardcoded paths to files, e.g. "tests/unit/sample_traces". Using the Path module is much cleaner imo

Reply:
What about using a fixture?

Reply:
I kept it here since the trace_path is tied to the agent_framework, which is a fixture already, so I think we're covered 👍